pkg/kine/server/get.go

```diff
@@ -5,18 +5,33 @@ import (
 	"fmt"
 
 	"go.etcd.io/etcd/api/v3/etcdserverpb"
+	"go.opentelemetry.io/otel/attribute"
 )
 
 func (l *LimitedServer) get(ctx context.Context, r *etcdserverpb.RangeRequest) (*RangeResponse, error) {
 	if r.Limit != 0 && len(r.RangeEnd) != 0 {
 		return nil, fmt.Errorf("invalid combination of rangeEnd and limit, limit should be 0 got %d", r.Limit)
 	}
 
+	ctx, span := tracer.Start(ctx, "backend.get")
```

Comment:
looks like you use both -  `backend.x` and `limited.x`. Any reason for this or should we settle on one version?

Reply:
Yes, so limited is the limitedserver.x call which then makes a call to backend.x, or backend.y based on the logic.

Reply:
![Screenshot from 2024-07-10 12-31-45](https://github.com/canonical/k8s-dqlite/assets/107889048/46c90751-4d75-4857-9e20-7772593ab548)